colabs/artifact_basics/Artifact_Basics.ipynb

```diff
@@ -0,0 +1,370 @@
+        "artifact.add_file(local_path=\"/content/sample_data/california_housing_test.csv\", name=\"sorted_file\")\n",
+        "run.log_artifact(artifact, aliases= [\"sorted\"]) # logs the new artifact version, overwriting the old one.\n",
+        "run.finish()\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Now the sorted file will be logged in `my_first_artifact`. Any changes you log to an artifact will overwrite any older version. \n",
+        "\n",
+        "The Artifact has also been given a custom `alias`, a label for this Artifact version. While the `alias` is currently `sorted`, the default aliases is `vN`, where `N` is the number of versions the Artifact has. This increments automatically."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "AyANijPlIN5m"
+      },
+      "source": [
+        "# Update Artifact version metadata"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "e4psfYzGeAVS"
+      },
+      "source": [
+        "You can update the `description`, `metadata`, and `alias` of an artifact on the W&B platform during or outside a W&B Run.\n",
+        "\n",
+        "\n",
+        "This example changes the `description` of the `my_first_artifact` artifact inside a run:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "16fCvZUZJD29"
+      },
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\")\n",
+        "artifact.description = \"This is an edited description.\"\n",
+        "artifact.save()  # persists changes to an Artifact's properties\n",
+        "run.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "o-rcMsZYh0Ig"
+      },
+      "source": [
+        "# Download an Artifact"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "B2ssTI7LrHS8"
+      },
+      "source": [
+        "To retrieve the path of an Artifact for external use, use the [`artifact.download()`](https://docs.wandb.ai/ref/python/artifact#download) method. This will retrieve the directory of an Artifact you select."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "18-7Kd9Oh8cl"
+      },
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\")\n",
+        "# This will download the specified artifact to where your code is running\n",
+        "datadir = artifact.download()\n",
+        "run.finish()\n",
```

Comment:
I would suggest moving the run.finish() to the end of the block as it is usually better practice (e.g., in this case, it would capture what is printed out)